conf/st2.full.conf

```diff
@@ -0,0 +1,182 @@
+[action_sensor]
+# Timeout value of all httprequests made by action sensor.
+request_timeout = 1
+# Amount of time to wait prior to retrying a request.
+retry_wait = 1
+# Whether to enable or disable the ability to post a trigger on action.
+enable = True
+# URL for action sensor to post TriggerType.
+triggers_base_url = http://localhost:9101/v1/triggertypes/
+# No. of times to retry registration.
+max_attempts = 10
+
+[actionrunner]
+# Python binary which will be used by Python actions.
+python_binary = /mnt/st2repos/st2/virtualenv/bin/python
+# location of the logging.conf file
+logging = conf/logging.conf
+
+[api]
+# List of origins allowed
+allow_origin = ['http://localhost:3000']
+# location of the logging.conf file
+logging = conf/logging.conf
+# Enable to serve static WebUI files using the API service
+serve_webui_files = True
+# StackStorm API server host
+host = 0.0.0.0
+# Send empty message every N seconds to keep connection open
+heartbeat = 25
+# StackStorm API server port
+port = 9101
+
+[api_pecan]
+# None
```

Comment:
Hm, seems about to add description for all the arguments which are missing it :)